### PR TITLE
SEO enhancement for the docs and update introduction section

### PR DESCRIPTION
**Pull request by IsAmrish**
Signed-off-by: isamrish <askmaurya48@gmail.com>

This PR will 
- Update SEO component
- Update content for the introduction section

### CODE REVIEW

File: docs/src/theme/Seo/index.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      {pageImage && <meta property="og:image:height" content={imageObj.shareImageHeight} /> }
+      
+      {keywords && (
+        <meta
+          name="keywords"
+          content={Array.isArray(keywords) ? keywords.join(',') : keywords}
+        />
+      )}
+      {/* Twitter Card */}
+      {pageImage && <meta name="twitter:card" content="summary_large_image" />}
```
Comment (rakeshPRaghu) [inline docs/src/theme/Seo/index.js:87]:
```suggestion
     <meta name="twitter:card" content="summary_large_image" />
```
even if the image is not present `twitter:card` is important

File: docs/versioned_docs/version-2.11.0/introduction/overview.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 id: overview
 title: OpenEBS Documentation
 slug: /
+keywords:
+  - OpenEBS
+  - OpenEBS overview
```
Comment (AVRahul) [inline docs/versioned_docs/version-2.11.0/introduction/overview.md:7]:
```suggestion
  - OpenEBS Overview
  - OpenEBS Documentation
```

File: docs/versioned_docs/version-2.11.0/introduction/community.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
 ---
 id: community
 title: OpenEBS Community
+keywords:
+  - Community
+  - OpenEBS Community
```
Comment (AVRahul) [inline docs/versioned_docs/version-2.11.0/introduction/community.md:6]: Keywords word case are different, for some both the starting letters are capital and some only the first word starts with a capital, would be better if we follow one approach throughout.

File: docs/src/theme/Seo/index.js (hunk 1 above)
Comment (IsAmrish) [reply docs/src/theme/Seo/index.js:87]: The image will always be present (If not provided markdown, it can come from defaultImage )

Comment (rakeshPRaghu) [reply docs/src/theme/Seo/index.js:87]: But the check here is not relevant here IMO

**Review by AVRahul — APPROVED** [review]
LGTM

**Review by rakeshPRaghu — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**